wiki/release/release-process.md

```diff
@@ -592,10 +592,14 @@ svn ls https://dist.apache.org/repos/dist/release/pulsar
 svn rm https://dist.apache.org/repos/dist/release/pulsar/pulsar-2.Y.0
 ```
 
-## 21. Move branch to next version
+## 20. Move branch to next version
 
-Follow the instructions in step 6, but run the script on the release branch.
+Run the following commands in release branches.
 
 ```shell
 ./src/set-project-version.sh 2.X.Y-SNAPSHOT
-```
\ No newline at end of file
+
+git commit -m 'Bumped version to 2.Y.0-SNAPSHOT' -a
+```
+
+For major releases, we need to create a Pull Request on GitHub. For patch releases, we can push the commit directly to the release branch.
```

Comment:
Major release version is done in Step 7.

Reply:
Got it.